### PR TITLE
[SW-1541] Fix get-extended-h2o script to reflect new location of sparkling water releases

### PR DESCRIPTION
**Pull request by jakubhava**

### CODE REVIEW

File: bin/get-extended-h2o.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ else
 fi
 
 h2odriver_file="h2odriver-$H2O_VERSION.$H2O_BUILD-${hadoop_version}extended.jar"
-h2odriver_url="${S3_RELEASE_BUCKET}/rel-${MAJOR_VERSION}/${PATCH_VERSION}/extended/${h2odriver_file}"
+h2odriver_url="${S3_RELEASE_BUCKET}/spark-${SPARK_MAJOR_VERSION}/${VERSION}/extended/${h2odriver_file}"
```
Comment (mn-mikke) [inline bin/get-extended-h2o.sh]: What about the path of nightly builds? Is it relevant?

Comment (jakubhava) [reply bin/get-extended-h2o.sh]: Good point

Comment (mn-mikke) [reply bin/get-extended-h2o.sh]: Yep, no problem ;-)

Comment (jakubhava) [reply bin/get-extended-h2o.sh]: thank you!

**Pull request merged**